gef.py

```diff
@@ -11166,6 +11167,8 @@ def __str__(self) -> str:
     def version(self) -> Optional[Tuple[int, int]]:
         if not is_alive():
             return None
+        if gef.config["gef.libc_version"] != "":
```

Comment:
Also check that `gef.libc.version` is not set here

Reply:
Got it. Will change the doc to something like "provide libc version when it cannot be detected instead of override". This solves @Grazfather 's concern as well, since whatever we found in dump will always takes precedence.

Reply:
I have updated the code and please let me know, if I didn't get your comment right.

Reply:
Maybe for later, but we might want a way to force a version. e.g. if gef is getting the version wrong it would be annoying if we tried to hint it but gef refused to listen.